dev/com.ibm.ws.security.saml.sso/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -178,7 +178,7 @@ allowCustomCacheKey=Customized cache key
 allowCustomCacheKey.desc=Allow generating a custom cache key to access the authentication cache and get the subject.
 
 spHostAndPort=SAML host name and port number
-spHostAndPort.desc=Specifies a SAML service provider host name and port number.
+spHostAndPort.desc=Specifies the host name and port number by which the idP addresses this SAML service provider.  This attribute might be necessary when the browser needs to be redirected to a router or proxy server instead of directly to the service provider.  The format for the value for this attribute is (scheme)://(proxyOrRouterHost):(proxyOrRouterPort).  For example, https://myRouter.com:443.
```

Comment:
Suggested edit:

Specifies the hostname and port number by which the idP addresses this SAML service provider. Use this attribute if the browser needs to be redirected to a router or proxy server instead of directly connecting to the service provider. The format for the value for this attribute is (scheme)://(proxyOrRouterHost):(proxyOrRouterPort). For example, https://myRouter.com:443

Reply:
@dmuelle Suggested updates have been pushed.